src/components/Button/Button.scss

```diff
@@ -34,6 +34,7 @@ $block: '.#{variables.$ns}button';
         color 0.15s linear;
     transform: scale(1);
     display: inline-flex;
+    flex-shrink: 0;
```

Comment:
That's not the cause of the problem. The problem is in the negative margins of `.g-button__icon` element. Setting `flex-shrink` to 0 would break user layouts relying on flex

Reply:
Oh, you're right
Fixed here 9047cd0f7c47b3a88bfe784fa7efa3b16fde5ac7
Test is here https://github.com/gravity-ui/uikit/pull/1579